deploy/docker-compose.yaml

```diff
@@ -1,19 +1,39 @@
-version: '3'
+version: "3"
 services:
   nuoadmin1:
     image: nuodb/nuodb-ce:latest
     environment:
       NUODB_DOMAIN_ENTRYPOINT: nuoadmin1
     hostname: nuoadmin1
-    command: [ "nuoadmin" ]
+    command: ["nuoadmin"]
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "nuocmd check servers --check-connected --check-converged --timeout 300",
+        ]
   sm:
     image: nuodb/nuodb-ce:latest
     hostname: sm
     environment:
       NUOCMD_API_SERVER: nuoadmin1:8888
     depends_on:
-      - nuoadmin1
-    command: [ "nuodocker", "start", "sm", "--db-name", "hockey", "--server-id", "nuoadmin1", "--dba-user", "dba", "--dba-password", "goalie" ]
+      nuoadmin1:
+        condition: service_healthy
+    command:
+      [
+        "nuodocker",
+        "start",
+        "sm",
+        "--db-name",
+        "hockey",
+        "--server-id",
+        "nuoadmin1",
+        "--dba-user",
+        "dba",
+        "--dba-password",
+        "goalie",
+      ]
```

Comment:
This is just a formatting change, except for the `condition: service_healthy` specified on the `nuoadmin1` dependency?

Reply:
Yes